### ISSUE TITLE
Issues linking Github Desktop to Atom...

### ISSUE THREAD

**Issue by JohnYingling** [Media-Ed-Online/intro-web-dev#13]
Having some serious problems here linking my Desktop Github to Atom.

<img width="1124" alt="screen shot 2017-09-08 at 1 03 25 pm" src="https://user-images.githubusercontent.com/31684100/30196852-4bf39c06-9496-11e7-9121-d73c66ee7457.png">

<img width="1430" alt="screen shot 2017-09-08 at 1 03 07 pm" src="https://user-images.githubusercontent.com/31684100/30196853-4bfd7550-9496-11e7-8c42-afe83e9f48fa.png">

Install seemed fine.  What on earth is going on here?

**Comment by JustineEvansUM** [comment]
Woah, @JohnYingling, what did you do? Crazy. 

First thing's first, try removing your Project folder from Atom. Simply right click and select 'Remove Project Folder.'

Next, let's try re-initializing over the top of your current GitHub Repo. In the GitHub App, right click on your repo folder and 'remove it.' Click through the warning message.

Next, select 'Add a local repository' and located your full repo path, like /john/homework/web-dev-hw. Press 'Fetch origin' at the top menu bar and see if you receive an error.

Then get back to me.

**Comment by JohnYingling** [comment]
That worked without blowing up on me....but...I mean, I got no glaring error, but I still feel like I'm missing something.

I don't have both the "uncommited changes" box you are talking of, or the option to "Publish Repo" and I'm not sure why.  Combed through everything multiple times. 

 Screens attached.

<img width="1354" alt="screen shot 2017-09-08 at 1 32 47 pm" src="https://user-images.githubusercontent.com/31684100/30197445-5826201c-949a-11e7-9f2b-2cf0d8f71e08.png">
<img width="1398" alt="screen shot 2017-09-08 at 1 30 33 pm" src="https://user-images.githubusercontent.com/31684100/30197447-58402002-949a-11e7-881d-faace8ebbfaf.png">
<img width="1357" alt="screen shot 2017-09-08 at 1 30 14 pm" src="https://user-images.githubusercontent.com/31684100/30197446-583fbde2-949a-11e7-8fee-43b429186f5b.png">

**Comment by JohnYingling** [comment]
Wait wait, did I already do this?

I actually made this on Github's site first...but did it connect?  
<img width="1395" alt="screen shot 2017-09-08 at 1 37 24 pm" src="https://user-images.githubusercontent.com/31684100/30197543-ec6ec9ea-949a-11e7-8371-804f6164fb54.png">

**Comment by JustineEvansUM** [comment]
I think you're moving in the right direction, for sure. Try making a change to file, like your README, or even creating an index.html (just through in a few characters) and save it. Make sure you save the files, or the App won't recognize changes.

**Comment by JustineEvansUM** [comment]
PS - I see a space in your screenshot filename... Watch out! That will be hard to link!

**Comment by JohnYingling** [comment]
@JustineEvansUM Why would it be hard to link?  Do you mean add to my Readme, or will I run into problems later?

Added to the readme already... will note this advice!  :) 

<img width="1214" alt="screen shot 2017-09-08 at 1 46 56 pm" src="https://user-images.githubusercontent.com/31684100/30197733-3a74fdca-949c-11e7-9dcd-eaf5690eedb4.png">

**Comment by JonSwallow** [comment]
@JohnYingling  You do have it connected according to your picture. To make a change you have to fill out the summary and description then you can commit. The button that says fetch origin will change to push to origin.
![example](https://user-images.githubusercontent.com/31550331/30198095-1e207a1e-9429-11e7-84a7-8b2f05e35b24.png)

**Comment by JustineEvansUM** [comment]
@JohnYingling and class, [here's an Issue](https://github.com/Media-Ed-Online/intro-web-dev/issues/20) from your classmate of where a space in a filename can cause problems!

Working spaces is largely a graphical interface perk of operating systems, but it's not allowable in all coding. For example, I work natively, building my sites in full before pushing them to a server. Here's a shot of my **CLI** or command-line interface (which is Terminal for macOS).
![Image of CLI](https://raw.githubusercontent.com/Media-Ed-Online/intro-web-dev/gh-pages/img/cli-file-names.jpg)
You'll see I can't change my directory to the folder "My Cool Files." It doesn't even recognize it - "No such file or directory." If I want to access the folder, I have to use a `\` to tell the system there's a space there. It gets really messy, believe me.

In other words, *don't use spaces.* You'll thank yourself later!

**Comment by mjohnson92** [comment]
This was really helpful, I had very similar issues just in different order. Thank you!

**Comment by tannercurey** [comment]
Thanks, I also found this issue helpful as I has similar problems.

**Comment by JohnYingling** [comment]
Sorted.
